LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/ChangeTrackingProperty.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace LoRaWan.NetworkServer
+{
+    using System;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Primitive that keep tracks of changes
+    /// </summary>
+    /// <typeparam name="T">The underlying type that we keep track of. Must implement <see cref="IEqualityComparer{T}"/></typeparam>
```

Comment:
I don't understand this comment. What needs to implement IEqualityComparer?

Reply:
T, the generic type. Is must implement equality comparer so I can flag as dirty.